<!DOCTYPE html>
<html lang="en">
    
    <head>
    <link href="https://gmpg.org/xfn/11" rel="profile">
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<!-- Enable responsiveness on mobile devices -->
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
<meta name="generator" content="Hugo 0.107.0">

    
    
    

<title>Archaeogenetics • open-archaeo</title>


<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Archaeogenetics"/>
<meta name="twitter:description" content="A list of open source archaeological software and resources"/>

<meta property="og:title" content="Archaeogenetics" />
<meta property="og:description" content="A list of open source archaeological software and resources" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://open-archaeo.info/tags/archaeogenetics/" /><meta property="og:site_name" content="Title" />



    


<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">








<link rel="stylesheet" href="/scss/hyde-hyde.0e8a8b00b5b21fefbbf03e7c07ce206d91ea5b85fc542939c12b078f2f4e92de.css" integrity="sha256-DoqLALWyH&#43;&#43;78D58B84gbZHqW4X8VCk5wSsHjy9Okt4=">


<link rel="stylesheet" href="/scss/print.2744dcbf8a0b2e74f8a50e4b34e5f441be7cf93cc7de27029121c6a09f9e77bc.css" integrity="sha256-J0Tcv4oLLnT4pQ5LNOX0Qb58&#43;TzH3icCkSHGoJ&#43;ed7w=" media="print">



<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">


<link rel="stylesheet" href="/css/custom.d18e24ca79a5ed85071e0272c25ad303c6f7cbe96e1d47f95f1e1332e87bf7ed.css" integrity="sha256-0Y4kynml7YUHHgJywlrTA8b3y&#43;luHUf5Xx4TMuh79&#43;0=">
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <!-- Icons -->
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
    <link rel="shortcut icon" href="/favicon.png">
    <link rel="alternate" type="application/rss+xml" href="https://open-archaeo.info/tags/archaeogenetics/index.xml" title="open-archaeo" />

    

</head>


    <body class=" ">
    
<div class="sidebar">
  <div class="container ">
    <div class="sidebar-about">
      <span class="site__title">
        <a href="https://open-archaeo.info/">
        
          open-archaeo
        
        </a>
      </span>
      
      
      <p class="site__description">
         A list of open source archaeological software and resources 
      </p>
    </div>
    <div class="collapsible-menu">
      <input type="checkbox" id="menuToggle">
      <label for="menuToggle">open-archaeo</label>
      <div class="menu-content">
        
<section class="sidebar-mainnav">
	<ul class="sidebar-nav">
		 
		
	</ul>
</section>
	

<div class="sidebar-taxonomy">
    <h2>Categories</h2>
	<ul class="sidebar-nav">
    
        <li>
            <a href="https://open-archaeo.info/categories/guides/" title="Entries in category 'Guides'" class="badge">GUIDES&nbsp;<span class="taxonomy-count post__meta">(30)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/categories/lists-and-datasets/" title="Entries in category 'Lists and datasets'" class="badge">LISTS AND DATASETS&nbsp;<span class="taxonomy-count post__meta">(73)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/categories/packages-and-libraries/" title="Entries in category 'Packages and libraries'" class="badge">PACKAGES AND LIBRARIES&nbsp;<span class="taxonomy-count post__meta">(230)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/categories/products/" title="Entries in category 'Products'" class="badge">PRODUCTS&nbsp;<span class="taxonomy-count post__meta">(15)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/categories/scripts/" title="Entries in category 'Scripts'" class="badge">SCRIPTS&nbsp;<span class="taxonomy-count post__meta">(69)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/categories/specifications-protocols-and-schemas/" title="Entries in category 'Specifications, protocols and schemas'" class="badge">SPECIFICATIONS, PROTOCOLS AND SCHEMAS&nbsp;<span class="taxonomy-count post__meta">(14)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/categories/standalone-software/" title="Entries in category 'Standalone software'" class="badge">STANDALONE SOFTWARE&nbsp;<span class="taxonomy-count post__meta">(102)</span></a>
        </li>
    
    </ul>
</div>


<div class="sidebar-taxonomy">
    <h2>Tags</h2>
	<ul class="sidebar-nav">
	
        <li>
            <a href="https://open-archaeo.info/tags/3d-modelling/" title="Entries tagged '3D modelling'" class="badge">3D modelling&nbsp;<span class="taxonomy-count post__meta">(19)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/adna-aligner/" title="Entries tagged 'aDNA aligner'" class="badge">aDNA aligner&nbsp;<span class="taxonomy-count post__meta">(1)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/adna-assembler/" title="Entries tagged 'aDNA assembler'" class="badge">aDNA assembler&nbsp;<span class="taxonomy-count post__meta">(2)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/adna-damage-aware-genotypers/" title="Entries tagged 'aDNA damage-aware genotypers'" class="badge">aDNA damage-aware genotypers&nbsp;<span class="taxonomy-count post__meta">(4)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/adna-kinship/" title="Entries tagged 'aDNA kinship'" class="badge">aDNA kinship&nbsp;<span class="taxonomy-count post__meta">(5)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/adna-preprocessing/" title="Entries tagged 'aDNA preprocessing'" class="badge">aDNA preprocessing&nbsp;<span class="taxonomy-count post__meta">(1)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/adna-simulators/" title="Entries tagged 'aDNA simulators'" class="badge">aDNA simulators&nbsp;<span class="taxonomy-count post__meta">(3)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/adna-toolkit/" title="Entries tagged 'aDNA toolkit'" class="badge">aDNA toolkit&nbsp;<span class="taxonomy-count post__meta">(1)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/aerial-and-satellite-imagery/" title="Entries tagged 'Aerial and satellite imagery'" class="badge">Aerial and satellite imagery&nbsp;<span class="taxonomy-count post__meta">(6)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/api-interfaces-and-web-scrapers/" title="Entries tagged 'API interfaces and web scrapers'" class="badge">API interfaces and web scrapers&nbsp;<span class="taxonomy-count post__meta">(20)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/archaeoastronomy/" title="Entries tagged 'Archaeoastronomy'" class="badge">Archaeoastronomy&nbsp;<span class="taxonomy-count post__meta">(3)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/archaeogenetics/" title="Entries tagged 'Archaeogenetics'" class="badge">Archaeogenetics&nbsp;<span class="taxonomy-count post__meta">(49)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/artefact-morphology/" title="Entries tagged 'Artefact morphology'" class="badge">Artefact morphology&nbsp;<span class="taxonomy-count post__meta">(11)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/augmented-reality/" title="Entries tagged 'Augmented reality'" class="badge">Augmented reality&nbsp;<span class="taxonomy-count post__meta">(2)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/bibliography/" title="Entries tagged 'Bibliography'" class="badge">Bibliography&nbsp;<span class="taxonomy-count post__meta">(4)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/biological-anthropology/" title="Entries tagged 'Biological anthropology'" class="badge">Biological anthropology&nbsp;<span class="taxonomy-count post__meta">(15)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/bits-and-bobs/" title="Entries tagged 'Bits and bobs'" class="badge">Bits and bobs&nbsp;<span class="taxonomy-count post__meta">(17)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/chronological-modelling/" title="Entries tagged 'Chronological modelling'" class="badge">Chronological modelling&nbsp;<span class="taxonomy-count post__meta">(40)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/cultural-evolution/" title="Entries tagged 'Cultural evolution'" class="badge">Cultural evolution&nbsp;<span class="taxonomy-count post__meta">(7)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/damage-profilers/" title="Entries tagged 'Damage profilers'" class="badge">Damage profilers&nbsp;<span class="taxonomy-count post__meta">(2)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/data-collection/" title="Entries tagged 'Data collection'" class="badge">Data collection&nbsp;<span class="taxonomy-count post__meta">(5)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/data-management/" title="Entries tagged 'Data management'" class="badge">Data management&nbsp;<span class="taxonomy-count post__meta">(33)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/datasets/" title="Entries tagged 'Datasets'" class="badge">Datasets&nbsp;<span class="taxonomy-count post__meta">(44)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/dendrochronology/" title="Entries tagged 'Dendrochronology'" class="badge">Dendrochronology&nbsp;<span class="taxonomy-count post__meta">(5)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/diagrams-and-visualizations/" title="Entries tagged 'Diagrams and visualizations'" class="badge">Diagrams and visualizations&nbsp;<span class="taxonomy-count post__meta">(14)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/drivers-and-io/" title="Entries tagged 'Drivers and IO'" class="badge">Drivers and IO&nbsp;<span class="taxonomy-count post__meta">(11)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/drones/" title="Entries tagged 'Drones'" class="badge">Drones&nbsp;<span class="taxonomy-count post__meta">(2)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/educational-resources-and-practical-guides/" title="Entries tagged 'Educational resources and practical guides'" class="badge">Educational resources and practical guides&nbsp;<span class="taxonomy-count post__meta">(39)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/ethics-and-professional-development/" title="Entries tagged 'Ethics and professional development'" class="badge">Ethics and professional development&nbsp;<span class="taxonomy-count post__meta">(4)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/games/" title="Entries tagged 'Games'" class="badge">Games&nbsp;<span class="taxonomy-count post__meta">(4)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/geoarchaeology/" title="Entries tagged 'Geoarchaeology'" class="badge">Geoarchaeology&nbsp;<span class="taxonomy-count post__meta">(3)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/geophysical-survey/" title="Entries tagged 'Geophysical survey'" class="badge">Geophysical survey&nbsp;<span class="taxonomy-count post__meta">(5)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/harris-matrix/" title="Entries tagged 'Harris matrix'" class="badge">Harris matrix&nbsp;<span class="taxonomy-count post__meta">(9)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/iconography/" title="Entries tagged 'Iconography'" class="badge">Iconography&nbsp;<span class="taxonomy-count post__meta">(4)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/instrumental-neutron-activation-analysis/" title="Entries tagged 'Instrumental Neutron activation analysis'" class="badge">Instrumental Neutron activation analysis&nbsp;<span class="taxonomy-count post__meta">(3)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/lidar/" title="Entries tagged 'LiDAR'" class="badge">LiDAR&nbsp;<span class="taxonomy-count post__meta">(2)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/lists/" title="Entries tagged 'Lists'" class="badge">Lists&nbsp;<span class="taxonomy-count post__meta">(19)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/literary-analysis-and-epigraphy/" title="Entries tagged 'Literary analysis and epigraphy'" class="badge">Literary analysis and epigraphy&nbsp;<span class="taxonomy-count post__meta">(6)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/lithic-analysis/" title="Entries tagged 'Lithic analysis'" class="badge">Lithic analysis&nbsp;<span class="taxonomy-count post__meta">(1)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/low-coverage-adapted-population-genetics-tools/" title="Entries tagged 'Low-coverage adapted population-genetics tools'" class="badge">Low-coverage adapted population-genetics tools&nbsp;<span class="taxonomy-count post__meta">(10)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/low-coverage-contamination-estimation/" title="Entries tagged 'Low-coverage contamination estimation'" class="badge">Low-coverage contamination estimation&nbsp;<span class="taxonomy-count post__meta">(5)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/luminescence-dating/" title="Entries tagged 'Luminescence dating'" class="badge">Luminescence dating&nbsp;<span class="taxonomy-count post__meta">(14)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/machine-learning/" title="Entries tagged 'Machine learning'" class="badge">Machine learning&nbsp;<span class="taxonomy-count post__meta">(2)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/metagenomic-classifers/" title="Entries tagged 'Metagenomic classifers'" class="badge">Metagenomic classifers&nbsp;<span class="taxonomy-count post__meta">(4)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/museums/" title="Entries tagged 'Museums'" class="badge">Museums&nbsp;<span class="taxonomy-count post__meta">(1)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/network-analysis/" title="Entries tagged 'Network analysis'" class="badge">Network analysis&nbsp;<span class="taxonomy-count post__meta">(1)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/palaeobotany/" title="Entries tagged 'Palaeobotany'" class="badge">Palaeobotany&nbsp;<span class="taxonomy-count post__meta">(21)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/palaeoenvironment-and-palaeobotany/" title="Entries tagged 'Palaeoenvironment and palaeobotany'" class="badge">Palaeoenvironment and palaeobotany&nbsp;<span class="taxonomy-count post__meta">(1)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/photogrammetry/" title="Entries tagged 'Photogrammetry'" class="badge">Photogrammetry&nbsp;<span class="taxonomy-count post__meta">(7)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/photography/" title="Entries tagged 'Photography'" class="badge">Photography&nbsp;<span class="taxonomy-count post__meta">(3)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/pipelines/" title="Entries tagged 'Pipelines'" class="badge">Pipelines&nbsp;<span class="taxonomy-count post__meta">(6)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/platforms-and-publications/" title="Entries tagged 'Platforms and publications'" class="badge">Platforms and publications&nbsp;<span class="taxonomy-count post__meta">(4)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/public-archaeology/" title="Entries tagged 'Public archaeology'" class="badge">Public archaeology&nbsp;<span class="taxonomy-count post__meta">(12)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/public-policy-and-civic-action/" title="Entries tagged 'Public policy and civic action'" class="badge">Public policy and civic action&nbsp;<span class="taxonomy-count post__meta">(2)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/radiocarbon-dating-calibration-and-sequencing/" title="Entries tagged 'Radiocarbon dating, calibration and sequencing'" class="badge">Radiocarbon dating, calibration and sequencing&nbsp;<span class="taxonomy-count post__meta">(34)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/schemas-and-ontologies/" title="Entries tagged 'Schemas and ontologies'" class="badge">Schemas and ontologies&nbsp;<span class="taxonomy-count post__meta">(21)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/seriation/" title="Entries tagged 'Seriation'" class="badge">Seriation&nbsp;<span class="taxonomy-count post__meta">(7)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/shape-recognition/" title="Entries tagged 'Shape recognition'" class="badge">Shape recognition&nbsp;<span class="taxonomy-count post__meta">(24)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/simulation/" title="Entries tagged 'Simulation'" class="badge">Simulation&nbsp;<span class="taxonomy-count post__meta">(8)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/site-mapping/" title="Entries tagged 'Site mapping'" class="badge">Site mapping&nbsp;<span class="taxonomy-count post__meta">(19)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/spatial-analysis/" title="Entries tagged 'Spatial analysis'" class="badge">Spatial analysis&nbsp;<span class="taxonomy-count post__meta">(29)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/stable-isotope-analysis/" title="Entries tagged 'Stable isotope analysis'" class="badge">Stable isotope analysis&nbsp;<span class="taxonomy-count post__meta">(11)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/statistical-analysis/" title="Entries tagged 'Statistical analysis'" class="badge">Statistical analysis&nbsp;<span class="taxonomy-count post__meta">(46)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/templates/" title="Entries tagged 'Templates'" class="badge">Templates&nbsp;<span class="taxonomy-count post__meta">(2)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/underwater-archaeology/" title="Entries tagged 'Underwater archaeology'" class="badge">Underwater archaeology&nbsp;<span class="taxonomy-count post__meta">(1)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/viewshed-analysis/" title="Entries tagged 'Viewshed analysis'" class="badge">Viewshed analysis&nbsp;<span class="taxonomy-count post__meta">(4)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/writing/" title="Entries tagged 'Writing'" class="badge">Writing&nbsp;<span class="taxonomy-count post__meta">(5)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/x-ray-fluorescence/" title="Entries tagged 'X-Ray Fluorescence'" class="badge">X-Ray Fluorescence&nbsp;<span class="taxonomy-count post__meta">(11)</span></a>
        </li>
    
        <li>
            <a href="https://open-archaeo.info/tags/zooarchaeology/" title="Entries tagged 'Zooarchaeology'" class="badge">Zooarchaeology&nbsp;<span class="taxonomy-count post__meta">(16)</span></a>
        </li>
    
    </ul>
</div>

        <section class="social">
	
	
	
	
	<a href="https://github.com/zackbatist/open-archaeo" rel="me"><i class="fab fa-github fa-lg" aria-hidden="true"></i></a>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
</section>

      </div>
    </div>
    
<div class="copyright">

  Maintained by Zack Batist </br>
  [<a href="https://github.com/zackbatist">GitHub</a>] [<a href="https://twitter.com/mtl_zack">Twitter</a>] </br>
  </br>
  
    <a href="https://creativecommons.org/publicdomain/zero/1.0/">CC0 1.0</a>
  
</div>


<div class="builtwith">
Built with <a href="https://gohugo.io">Hugo</a> ❤️ <a href="https://github.com/htr3n/hyde-hyde">hyde-hyde</a>.
</div>


  </div>
</div>

        <div class="content container">
            
    <h2 class="section__title">Archaeogenetics</h2>


    
<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/adrsm/">adrsm</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A tool designed to simulate the paired-end sequencing of a metagenomic community.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>maxibor</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/maxibor/adrsm">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-simulators">adna simulators</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/ancientmetagenomedir/">AncientMetagenomeDir</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Repository containing lists of all published ancient metagenome samples (and related).</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>SPAAM-community</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/lists-and-datasets">LISTS AND DATASETS</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/SPAAM-community/AncientMetagenomeDir">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/palaeobotany">palaeobotany</a>
           
        
          <a class="badge badge-tag" href="/tags/lists">lists</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/anfo/">anfo</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>The ANFO ancient DNA mapper.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>mpieva</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/mpieva/anfo">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-aligner">adna aligner</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/angsd/">angsd</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Program for analysing next generation sequencing data.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>ANGSD</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/ANGSD/angsd">github</a>
        
          <a class="badge badge-category" href="http://www.popgen.dk/angsd/index.php/ANGSD">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-contamination-estimation">low-coverage contamination estimation</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/antcaller/">AntCaller</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>SNP calling and genotyping for damaged DNA.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>BoyanZhou</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/BoyanZhou/AntCaller">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-damage-aware-genotypers">adna damage-aware genotypers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/archaeodiet/">archaeodiet</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Pipeline for metagenomic identification and authentication of ancient dietary DNA from archaeological samples.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>jfy133</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/jfy133/archaeodiet">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/ariadna/">ARIADNA</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Machine Learning Method for Ancient DNA Variant Discovery.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>Joseph K. Kawash</strong>,
        
          
          
          <strong>Andrey Grigoriev</strong>,
        
          
          
          <strong>Sean Smith</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://osf.io/5bph4/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-damage-aware-genotypers">adna damage-aware genotypers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/atlas/">atlas</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A set of tools that cover all programs necessary to obtain variant calls, estimates of heterozygosity and more from a BAM file.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>wegmannlab</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://bitbucket.org/wegmannlab/atlas/wiki/Home">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/pipelines">pipelines</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/authentict/">AuthentiCT</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Estimate the proportion of present-day DNA contamination in ancient DNA datasets generated from single-stranded libraries.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>StephanePeyregne</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/StephanePeyregne/AuthentiCT">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-contamination-estimation">low-coverage contamination estimation</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/consensify/">Consensify</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Consensify method for error reduction in ancient DNA pseudohaploidisation.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>jlapaijmans</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/jlapaijmans/Consensify">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/contamld/">ContamLD</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Software designed to estimate autosomal contamination in ancient DNA samples.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>nathan-nakatsuka</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/nathan-nakatsuka/ContamLD">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-contamination-estimation">low-coverage contamination estimation</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/coproid/">coproID</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>CoproID helps you to identify the &ldquo;true maker&rdquo; of Illumina sequenced Coprolites/Paleofaeces by checking the microbiome composition and the endogenous DNA.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>maxibor</strong>,
        
          
          
          <strong>nf-core</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/maxibor/coproID">github</a>
        
          <a class="badge badge-category" href="https://coproid.readthedocs.io/en/latest/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/palaeobotany">palaeobotany</a>
           
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/pipelines">pipelines</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/cuperdec/">cuperdec</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>R package to generate &lsquo;Cumulative Percent Decay&rsquo; curves, with optional filtering functions, for microbial taxonomic profiles.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>jfy133</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://cran.r-project.org/web/packages/cuperdec/index.html">cran</a>
        
          <a class="badge badge-category" href="https://github.com/jfy133/cuperdec">github</a>
        
        <br />
      
    
    
    
    
      <i class="ai ai-doi"></i> <a href="https://doi.org/10.5281/zenodo.4561901">10.5281/zenodo.4561901</a>
      <br />
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/damageprofiler/">DamageProfiler</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A Java based tool to determine damage patterns on ancient DNA as a replacement for mapDamage.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>JudithNeukamm</strong>,
        
          
          
          <strong>apeltzer</strong>,
        
          
          
          <strong>jfy133</strong>,
        
          
          
          <strong>alexhbnr</strong>,
        
          
          
          <strong>Integrative-Transcriptomics</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/Integrative-Transcriptomics/DamageProfiler">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/damage-profilers">damage profilers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/dammet/">DamMet</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A full probabilistic model for mapping ancient methylomes.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>KHanghoj</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/KHanghoj/DamMet">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/dates/">DATES</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Method for dating admixture in ancient DNA specimens.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>priyamoorjani</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/priyamoorjani/DATES">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/dice/">dice</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Joint estimate of demography and contamination for nuclear ancient DNA.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>grenaud</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/grenaud/dice">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-contamination-estimation">low-coverage contamination estimation</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/eager/">eager</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A fully reproducible and state-of-the-art ancient DNA analysis pipeline.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>nf-core</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/nf-core/eager">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/eager-cli/">EAGER-CLI</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A fully reproducible and state-of-the-art ancient DNA analysis pipeline.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>apeltzer</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/apeltzer/EAGER-CLI">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/pipelines">pipelines</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/eager-gui/">EAGER-GUI</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A fully reproducible and state-of-the-art ancient DNA analysis pipeline.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>apeltzer</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/apeltzer/EAGER-GUI">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/pipelines">pipelines</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/epipaleomix/">epiPALEOMIX</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A fast, accurate, and automatic pipeline for generating nucleosome and methylation maps from high throughput sequencing data underlying ancient samples.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>KHanghoj</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/KHanghoj/epiPALEOMIX">github</a>
        
          <a class="badge badge-category" href="https://bitbucket.org/khanghoj/epipaleomix/wiki/Home">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/pipelines">pipelines</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/gargammel/">gargammel</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A set of programs aimed at simulating ancient DNA fragments. For ancient hominin samples our program can also simulate various levels of present-day human contamination and microbial contamination.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>grenaud</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/grenaud/gargammel">github</a>
        
          <a class="badge badge-category" href="https://grenaud.github.io/gargammel/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-simulators">adna simulators</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/grups/">grups</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A collection of Python and R scripts that can be used to simulate the pairwise genetic distances between human individuals in a provided test pedigree.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>sameoldmike</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/scripts">SCRIPTS</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/sameoldmike/grups/">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-kinship">adna kinship</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/haproh/">hapROH</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Software to call ROH from ancient and present-day DNA using reference haplotypes.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>hringbauer</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/hringbauer/hapROH">github</a>
        
          <a class="badge badge-category" href="https://pypi.org/project/hapROH/">pypi</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/hops/">HOPS</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A pipeline which focuses on screening MALT data for the presence of a user-specified list of target species.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>rhuebler</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/rhuebler/HOPS">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/metagenomic-classifers">metagenomic classifers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/leehom/">leeHom</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Bayesian reconstruction of ancient DNA fragments.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>grenaud</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/grenaud/leeHom">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-preprocessing">adna preprocessing</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/madam/">MADAM</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Two-level de novo assembly strategy for ancient bacterial genomes.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>Integrative-Transcriptomics</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/Integrative-Transcriptomics/MADAM">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-assembler">adna assembler</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/malt/">MALT</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A sequence alignment and analysis tool designedfor processing high-throughput sequencing data, especially in the context of metagenomics.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>Daniel H. Huson</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://software-ab.informatik.uni-tuebingen.de/download/malt/welcome.html">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/metagenomic-classifers">metagenomic classifers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/mapdamage/">mapDamage</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Tracking and quantifying damage patterns in ancient DNA sequences.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>ginolhac</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/ginolhac/mapDamage">github</a>
        
          <a class="badge badge-category" href="http://ginolhac.github.io/mapDamage/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/damage-profilers">damage profilers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/mapping-iterative-assembler/">mapping-iterative-assembler</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Consensus calling (or &ldquo;reference assisted assembly&rdquo;), chiefly of ancient mitochondria.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>mpieva</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/mpieva/mapping-iterative-assembler">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-assembler">adna assembler</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/maximum-likelihood-relatedness-estimation/">maximum-likelihood-relatedness-estimation</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Infer biological relatedness from low coverage 2nd generation sequencing data.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>COMBINE-lab</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/COMBINE-lab/maximum-likelihood-relatedness-estimation">github</a>
        
          <a class="badge badge-category" href="https://github.com/COMBINE-lab/maximum-likelihood-relatedness-estimation/wiki">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-kinship">adna kinship</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/metabit/">metabit</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>An integrative and automated metagenomic pipeline for analysing microbial profiles from high-throughput sequencing shotgun data.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>Glouvel</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://bitbucket.org/Glouvel/metabit/src/master/">bitbucket</a>
        
          <a class="badge badge-category" href="https://bitbucket.org/Glouvel/metabit/wiki/Home">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/metagenomic-classifers">metagenomic classifers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/multivcfanalyzer/">MultiVCFAnalyzer</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A SNP filtering and SNP alignment generation tool, designed around (but not limited to) low coverage ancient DNA data.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>alexherbig</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/alexherbig/MultiVCFAnalyzer">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/ngsrelate/">NgsRelate</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Infer relatedness, inbreeding coefficients and many other summary statistics for pairs of individuals.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>ANGSD</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/ANGSD/NgsRelate">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-kinship">adna kinship</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/paleomix/">paleomix</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Pipelines and tools for the processing of ancient and modern High-Throughput Sequencing (HTS) data.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>MikkelSchubert</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/MikkelSchubert/paleomix/">github</a>
        
          <a class="badge badge-category" href="https://paleomix.readthedocs.io/en/latest/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/pipelines">pipelines</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/pathphynder/">pathPhynder</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A workflow for integrating ancient lineages into present-day phylogenies.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>ruidlpm</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/ruidlpm/pathPhynder/">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/pileupcaller/">pileupcaller</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Sample alleles from low coverage sequence data.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>stschiff</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/stschiff/sequenceTools">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-damage-aware-genotypers">adna damage-aware genotypers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/pmdtools/">PMDtools</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Compute postmortem damage patterns and decontaminate ancient genomes.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>pontussk</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/pontussk/PMDtools">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/poseidon-framework/">poseidon-framework</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Poseidon is a framework that seeks to provide standardized ways to store and share archaeogenetic, and perhaps more broadly, molecular archaeological datasets.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>poseidon-framework</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/poseidon-framework/poseidon-hs">github</a>
        
          <a class="badge badge-category" href="https://poseidon-framework.github.io/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/schemas-and-ontologies">schemas and ontologies</a>
           
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/read/">READ</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A method to infer the degree of relationship (up to second degree, i.e. nephew/niece-uncle/aunt, grandparent-grandchild or half-siblings) for a pair of low-coverage individuals.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>tguenther</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://bitbucket.org/tguenther/read/src/master/">bitbucket</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-kinship">adna kinship</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/schmutzi/">schmutzi</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Maximum a posteriori estimate of contamination for ancient samples.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>grenaud</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/grenaud/schmutzi">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-contamination-estimation">low-coverage contamination estimation</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/sex-deterrmine/">Sex.DetERRmine</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Calculate the relative coverage of X and Y chromosomes, and their associated error bars, from the depth of coverage at specified SNPs.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>TCLamnidis</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/scripts">SCRIPTS</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/TCLamnidis/Sex.DetERRmine">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/low-coverage-adapted-population-genetics-tools">low-coverage adapted population-genetics tools</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/snpad/">snpAD</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Implements an iterative method to jointly estimate error rates and genotypes frequencies from the data.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>Kay Prüfer</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://bioinf.eva.mpg.de/snpAD/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-damage-aware-genotypers">adna damage-aware genotypers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/sparse/">SPARSE</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Indexes &gt;100,000 reference genomes in public databases in to hierarchical clusters and uses it to predict origins of metagenomic reads.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>zheminzhou</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/zheminzhou/SPARSE">github</a>
        
          <a class="badge badge-category" href="https://sparse.readthedocs.io/en/latest/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/metagenomic-classifers">metagenomic classifers</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/statuaar/">statuAAR</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Estimation of human stature from long bone measurements according to several well established estimation formula.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>chrinne</strong>,
        
          
          
          <strong>capsizingHR</strong>,
        
          
          
          <strong>AnKaLoy</strong>,
        
          
          
          <strong>FrankSiegmund</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/ISAAKiel/statuAAR">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/tapas/">tapas</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>Evaluate the effectiveness of any short read mapper and its parameters using artificially generated reads.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>mlell</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/mlell/tapas">github</a>
        
          <a class="badge badge-category" href="https://mlell.github.io/tapas/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-simulators">adna simulators</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/tkgwv2/">tkgwv2</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>An ancient DNA relatedness pipeline for ultra-low coverage whole genome shotgun data.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>danimfernandes</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/danimfernandes/tkgwv2">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-kinship">adna kinship</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/topas/">TOPAS</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>A toolkit for efficient manipulation of sequence data in various ways.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>subwaystation</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/standalone-software">STANDALONE SOFTWARE</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/subwaystation/TOPAS">github</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
           
        
          <a class="badge badge-tag" href="/tags/adna-toolkit">adna toolkit</a>
          
        
        <br />
      
    
</div>


  
</article>

<article class="openarchaeo-item">
  <header>
    <span class="openarchaeo-item-name"><a href="/post/yada/">yada</a></span>
    <span class="openarchaeo-item-description"><!-- Generated by csv2md.R – do not edit by hand -->
<p>yada, which stands for Yet Another Demographic Analysis package, supports a variety of demographic tasks, with most applications drawn from forensic anthroplogy and archaeology.</p>
</span>
  </header>
  
  
<div class="post__meta">
    
    
    
    
    
      
      
        by
        
        
          
          
          <strong>MichaelHoltonPrice</strong>
        
        <br />
      
    
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/packages-and-libraries">PACKAGES AND LIBRARIES</a>
              
          
          <br />
      
    
    
    
    
      
      
        <i class="fas fa-link"></i>
        
          <a class="badge badge-category" href="https://github.com/eehh-stanford/yada">github</a>
        
          <a class="badge badge-category" href="https://eehh-stanford.github.io/yada/">website</a>
        
        <br />
      
    
    
    
    
    
    
    
      
      
         <i class="fas fa-tags"></i>
        
          <a class="badge badge-tag" href="/tags/archaeogenetics">archaeogenetics</a>
          
        
        <br />
      
    
</div>


  
</article>





        </div>
        
    
<script defer src="https://use.fontawesome.com/releases/v5.12.1/js/all.js" integrity="sha384-ZbbbT1gw3joYkKRqh0kWyRp32UAvdqkpbLedQJSlnI8iLQcFVxaGyrOgOJiDQTTR" crossorigin="anonymous"></script>



    </body>
</html>
